scripts/transition_metrics.py

```diff
@@ -43,6 +72,7 @@ def rx_commission_decommission(filename, non_lwr):
         names of nonreactor facilities in the simulation
 
     Returns:
+    --------
     simulation_data: DataFrame
         Reactor data of the simulation with the additional
```

Comment:
Not sure if this dataframe is reading in simulation results or feeding into the simulation itself. If the latter, consider the following suggestion:
```suggestion
        Reactor data for the simulation with the additional
```